src/algebra/linear_ordered_comm_group_with_zero.lean

```diff
@@ -171,7 +171,8 @@ lemma div_le_div' (a b c d : α) (hb : b ≠ 0) (hd : d ≠ 0) :
 begin
   by_cases ha : a = 0, { simp [ha] },
   by_cases hc : c = 0, { simp [inv_ne_zero hb, hc, hd], },
-  exact @div_le_div_iff' _ _ (units.mk0 a ha) (units.mk0 b hb) (units.mk0 c hc) (units.mk0 d hd)
+  exact
+    @div_le_div_iff' _ (units.mk0 a ha) (units.mk0 b hb) (units.mk0 c hc) (units.mk0 d hd) _ _ _
```

Comment:
Having the typeclass arguments after the variables like this is very unusual - can you change things so that the first three `_`s are on the left?

Reply:
I re-ordered the variables.  This means that each section redefines the variables {a b c d}: let me know if there is a way of having some assumptions "float" to the last position automatically!